### DISCUSSION TITLE
Is LTMeat a soft kill or hard kill?

### DISCUSSION THREAD

**Discussion by KandyKaneMarvel** [3kh0/ext-remover#681] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I've been thinking about using it but if it's a hard kill it could bring trouble.
Much appreciated,
EcoliEater87

**Comment by ghost** [comment, marked as answer]
I tested it, and it is definitely a soft kill. just close the hanging page.
